src/compas_fab/ghpython/components/compile.py

```diff
@@ -0,0 +1,24 @@
+import clr
+
+# The below line does not work, since Grasshopper cannot be loaded from withing RhinoPython Editor.
+#from compas_fab.ghpython.components import filename
+
```

Comment:
then maybe just remove it and also the comment. There's a bit of duplication but is not too bad, and the comment makes it feel like if something is broken and needs fixing.